sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -505,6 +505,9 @@ def __init__(self, options, proto_pipeline):
           self.proto.transformNameMapping.additionalProperties.append(
               dataflow.Job.TransformNameMappingValue.AdditionalProperty(
                   key=key, value=value))
+    if self.google_cloud_options.create_from_snapshot:
```

Comment:
What is the option name in other SDKs? There is a minor mismatch between `createdFromSnapshotId` and `create_from_snapshot`. Is id relevant here?

Reply:
This matched java https://github.com/apache/beam/blob/master/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java#L1181